build.rs

```diff
@@ -62,26 +62,30 @@ fn generate_tests(out_dir: &OsStr) {
         let path = path.unwrap();
         let path = path.path();
 
-        let name = path.file_stem().unwrap().to_str().unwrap();
+        let name = path.file_stem().unwrap().to_str().unwrap().to_lowercase();
 
         string.push_str(&format!(r#"
         #[test]
         fn {0}() {{
             let mut languages = Languages::new();
             languages.get_statistics(vec!["{1}"], Vec::new());
+
+            let languages = languages.into_iter().collect::<Vec<_>>();
+            assert_eq!(languages.len(), 1, "wrong number of languages found");
```

Comment:
This error message should be clearer. Something like `Expected just cpp, found: ...`